lib/routes/bjnews/cat.ts

```diff
@@ -0,0 +1,58 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import { load } from 'cheerio';
+import ofetch from '@/utils/ofetch';
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+
+export const route: Route = {
+    path: '/:cat',
+    categories: ['traditional-media'],
+    example: '/bjnews/depth',
+    parameters: { cat: '分类, 可从URL中找到' },
+    features: {},
+    radar: [
+        {
+            source: ['www.bjnews.com.cn/:cat'],
+        },
+    ],
+    name: '分类',
+    maintainers: ['dzx-dzx'],
+    handler,
+    url: 'www.bjnews.com.cn',
+};
+
+async function handler(ctx) {
+    const url = `https://www.bjnews.com.cn/${ctx.req.param('cat')}`;
+    const res = await ofetch(url);
+    const $ = load(res);
+    const list = $('#waterfall-container .pin_demo > a')
+        .toArray()
+        .slice(0, ctx.req.query('limit') ? Number.parseInt(ctx.req.query('limit')) : 15)
+        .map((a) => ({
+            title: $(a).text(),
+            link: $(a).attr('href'),
+            category: [`source:${$(a).parent().find('.source').text().trim()}`],
```

Comment:
```suggestion
            category: $(a).parent().find('.source').text().trim(),
```

Reply:
I want to distinguish it with each page's own keywords, if there would be.

Reply:
...No?